tools/emptyDropsWrapper/README.md

```diff
@@ -0,0 +1,15 @@
+# emptyDropsWrapper
+
+Reads in an rds file containing the count matrix in dgCMatrix format in gene x droplet orientation and produces a CSV file indicating cell filtering and other cell metadat as returned by the emptyDrops function.
+
+## Requirements
+
+This script requires the following R packages to be installed: 'optparse', 'DropletUtils'.
```

Comment:
nit: use bullets to make it more conspicuous:
- `optparse`
- `DropletUtils`

Reply:
done